readme.md

```diff
@@ -276,10 +276,10 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
   | Package | GitHub Branch | LTS Tag | LTS Start Date | Maintenance End Date |
   | :-----: | :-----------: | :-----: | :------------: | :------------------: |
+  | vcpkg: 2023-07-27 | lts_07_2023 | LTS_07_2023_Ref01 | 2023-07-27 | 2024-07-27 |
   | vcpkg: 2023-04-20 | lts_01_2023 | LTS_01_2023_Ref02 | 2023-04-20 | 2024-04-20 |
   | vcpkg: 2023-01-13 | lts_01_2023 | LTS_01_2023_Ref01 | 2023-01-13 | 2024-01-13 |
   | vcpkg: 2022-09-15 | lts_07_2022 | LTS_07_2022_Ref02 | 2022-09-15 | 2023-09-15 |
-  | vcpkg: 2022-07-18 | lts_07_2022 | LTS_07_2022_Ref01 | 2022-07-18 | 2023-07-18 |
```

Comment:
I think it is useful to have a history of all LTSs and when the support ended.

Reply:
Our regular procedure is to remove any entries in this table once they pass their end date.